docs/course/course.yaml

```diff
@@ -190,3 +190,66 @@ paths:
                 status: 404
                 code: DOCUMENT_NOT_FOUND
                 message: Course with the specified id was not found.
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Courses
+      summary: Find one course.
```

Comment:
find course by id
you changed name of endpoint, so change this too

Reply:
> find course by id you changed name of endpoint, so change this too

fixed it